test/recipes/30-test_evp_data/evppkey_rsa_common.txt

```diff
@@ -2026,3 +2029,34 @@ Ctrl = digest:SHA256
 Ctrl = rsa_padding_mode:x931
 Input = "0123456789ABCDEF123456789ABCDEFG"
 Output = 4b75f521e2e6eeda3f2dcb84ebdacbadeab8ffc1ecdf06c7c4e38727e082a8f5e71be66cdee6d14da1d3a0f18ff20914f71b5308363c81e7471688f4f201e82603ea6a7f31da89cd846b30e2893fd956e76b3d23d40f733e0358e3883526158c74577ba43cc664eaeb909818e75b89fc764cf4575517f87251f8d3cf29b1532f33e6183d454bddd6f255d0ca4415d957eb90dbc55d047f48a01c6e68d5ab46327a158ff7a3383b5b0446b8cd4a91b8859abd3285020a1613ef17d3f8562147828bb36be65505eeec77e968d04e172e2851ff1d7ef523b4022deb72d5fbf78db6738d170098586b904d1c369cedb5e3fe1b909a8dd8ac3beb521af2510d044580
+
+# RSA signing with PSS salt length >= digest length is not approved in FIPS 186-5
+FIPSversion = >= 3.4.0
+Sign = RSA-PSS-DEFAULT
+Ctrl = rsa_pss_saltlen:30
```

Comment:
Should this check the -1, -2,-3, -4 cases also?

Reply:
Updated. The following cases are tested: fixed salt length, "digest", "max", "auto" and "auto-digestmax".